app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,6 @@
+<% provide(:title, 'Contact') %>
+<h1>Contact</h1>
+<p>
+  Contact the Ruby on Rails Tutorial about the sample app at the
+  <%= link_to "contact page.", "http://www.railstutorial.org/contact" %>
```

Comment:
Please use I18n for multi-language support

Reply:
done